public/apps/configuration/utils/display-utils.tsx

```diff
@@ -124,3 +126,11 @@ export function ExternalLinkButton(props: { text: string; href: string } & EuiBu
     </EuiButton>
   );
 }
+
+export function ExternalLink(props: { href: string }) {
+  return (
+    <EuiLink external={true} href={props.href} target="_blank">
+      {LEARN_MORE}
+    </EuiLink>
+  );
+}
```

Comment:
Q: Are we able to handle the external icon itself in a separate row?
![image](https://user-images.githubusercontent.com/63078162/92650491-72eccf00-f2a1-11ea-9949-26d9d683ce45.png)

Reply:
It cannot. I just added "display: inline-block" and noticed that it remains as a block during window dragging.